lib/Platform/Intl/java/com/facebook/hermes/intl/PlatformDateTimeFormatterAndroid.java

```diff
@@ -15,10 +15,17 @@
 import java.util.Date;
 import java.util.Locale;
 import java.util.TimeZone;
+import java.util.Optional;
+import java.util.List;
+import java.util.Arrays;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
 
 public class PlatformDateTimeFormatterAndroid implements IPlatformDateTimeFormatter {
   private DateFormat mDateFormat = null;
 
+  private List<String> allAvailableTimeZones = Arrays.asList(TimeZone.getAvailableIDs());
```

Comment:
We probably don't need to maintain this list, since we can just call `getAvailableIDs` whenever we need the IDs.

Reply:
Remember to delete this